Add Ultrasonic Subsystem

wh1ter0se opened the pull request:
Added ultrasonic commands for positioning subsystem (sorry if this is done incorrectly)

wh1ter0se committed:
Merge pull request #2 from RedInquisitive/master

UPDATE from master to wh1ter0se fork

wh1ter0se committed:
Create SubsystemPositioning.java

wh1ter0se committed:
Update SubsystemPositioning.java

kbowen99 commented:
Viewing this on my phone, it seems there isn't any code just an import statement for a positioning subsystem and no command either. You may want to push your local branch, and make sure all your code is online

wh1ter0se committed:
Format Setup

Added command import
Added package formatting

wh1ter0se committed:
Added CommandUltrasonicRange

Still a lot to figure out with the input from the analog ultrasonic sensor but the setup done
